Apps/AdminWebClient/src/ClientApp/src/components/core/CommunicationTable.vue

```diff
@@ -18,8 +18,8 @@ import { ICommunicationService } from "@/services/interfaces";
 @Component({
     components: {
         BannerModal,
-        EmailModal,
-    },
+        EmailModal
```

Comment:
I prefer when a trailing comma is included on the final list item. It makes additions and removals to lists less noisy in the diffs.

Reply:
Also, prettier enforces the extra commas.  You should run `npx prettier --write . '!**/{bin,obj,dist,sonar_reports}/**'` from the root directory to auto-resolve this.

Reply:
Does that mean that running the linter locally vs the root directory have different outcomes?

Reply:
Yes, specifically for the Admin_ClientApp project.  I didn't end up figuring out why it behaved differently.

Reply:
That might be problematic. If anything, the local settings should trump the higher ones

Reply:
I'm not sure why this happened, looking into it, it seems to that i happened only on AdminWebClient and not in WebClient

Reply:
It was missing the prettier library in adminwebclient

Reply:
I tried that at the time and still had some issues.  I'm not sure if it's specific to my machine or not.  I originally posted about it [here](https://github.com/bcgov/healthgateway/pull/2144#discussion_r606164832):

> Yes. I did some digging and there seems to be something wrong with the prettier formatting in the AdminWebClient ClientApp. It removes braces from inline functions as seen above. Also, it removes the trailing comma on the last item in imports and array/object declarations and doesn't format those lists properly if you mess with the whitespace/newlines.
> 
> I wasn't able to figure out a way to resolve the underlying issue, but if you run prettier from a directory higher than ClientApp, it will apply the appropriate formatting. (e.g. `npx prettier -w . '!**/{bin,obj,dist,sonar_reports}/**'`)
> 
> However, if you run the linter in the project or save one of the affected files in VSCode (with formatOnSave enabled), the bad formatting will be reinstated.

The best lead I had was that when I run the same `npx prettier --write . '!**/{bin,obj,dist,sonar_reports}/**'` via command-line from the AdminWebClient directory, it gives an error, while it works fine in WebClient or in a parent directory.

Reply:
Per our discussion, the addition of the prettier library seems to have fixed it, so I'll have to investigate separately what's going wrong on my machine.